### DISCUSSION TITLE
Load cases run in OC4 Phase II

### DISCUSSION THREAD

**Discussion by icnosvbovwb** [OpenFAST/openfast#1548] [Q&A]
Dear @jjonkman 
![image](https://user-images.githubusercontent.com/107483730/234567730-e5a375c0-93b5-472a-8fc6-f41188fccab7.png)
![image](https://user-images.githubusercontent.com/107483730/234567930-cacf86f8-da84-429b-af22-d64b43d9d3eb.png)

May I ask what the meaning of the Support structure in Table 3: Load cases run in OC4 Phase II is? In the report, it is mentioned that for these simulations, the platform, moonings, and tower are flexible, but the nacelle, drivetrain, and rotor are rigid and the generator is locked. 
Q1: But do I need to set False in the blade section of DEGREES OF FREEDOM? Or is the setting displayed in the second image correct?

Q2: If it is a Platform and Mooring. Is the setting displayed in the picture correct?
![image](https://user-images.githubusercontent.com/107483730/234569001-831164a9-e1b0-498c-9de0-4b2896bd06d4.png)

**Comment by jjonkman** [comment]
Dear @icnosvbovwb,

The first of your images is correct.  For this set of load cases, you want the tower and platform degrees of freedom enabled, but you want nacelle, drivetrain, and blade degrees of freedom disabled.

Best regards,

**Comment by icnosvbovwb** [comment]
Dear @jjonkman 
![image](https://user-images.githubusercontent.com/107483730/234574854-b37f7189-f48a-4a62-a3be-960f1692cb00.png)
![image](https://user-images.githubusercontent.com/107483730/234575016-183702dc-bd32-42bf-8ffd-0f90beaa287d.png)
If this is the case, is the tower part set to False?

**Reply by jjonkman** [reply]
I agree that the tower degrees of freedom should be disabled for the IEA Wind Task 30 OC4 Phase II 1.3* cases.

Best regards,
